LocalSettings.php

```diff
@@ -1213,6 +1213,24 @@
 		'default' => 'en',
 	],
 
+	// DiscussionTools
+	'wgDiscussionTools_visualenhancements' => [
+		'default' => 'default',
+		'isvwiki' => 'available',
+	],
+	'wgDiscussionTools_visualenhancements_reply' => [
+		'default' => 'default',
+		'isvwiki' => 'available',
+	],
+	'wgDiscussionTools_visualenhancements_pageframe' => [
+		'default' => 'default',
+		'isvwiki' => 'available',
+	],
+	'wgDiscussionTools_visualenhancements_newsectionlink_enable' => [
```

Comment:
Looks like this setting isn't currently available in the version of the extension we're using. We'd need to update to master branch (or REL1_40) to support it. Additionally, it appears that [the default value is true](https://github.com/wikimedia/mediawiki-extensions-DiscussionTools/blob/6ea3c7099dbec2ff7e6063b7dc5291ae517d3486/extension.json#L570-L573). WMF probably have it as false in their settings to phase in the feature.

Reply:
Removed it.